CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ function (superbuild_find_projects var)
     cxx11
     draco
     embree
+    exodus
```

Comment:
exodus is not a project, that cant work

Reply:
`egl` is not a project but it works somehow...

Reply:
it is of course, in the common superbuild

Reply:
as is cxx11 and all things listed here

Reply:
should work then!